develop/api_metadata.rst

```diff
@@ -110,6 +110,40 @@ The shardstorage column in pg_dist_shard indicates the type of storage used for
 |                |                      | | distributed file_fdw tables)                                              |
 +----------------+----------------------+-----------------------------------------------------------------------------+
 
+.. _citus_shards:
+
+Shard information view
+~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to the low-level shard metadata table described above, Citus provides a ``citus_shards`` view to easily check:
+
+* Where each shard is (node, and port),
+* What kind of table it belongs to, and
+* Its size
+
+This view helps you inspect shards to find, among other things, any size imbalances across nodes.
+
+.. code-block:: sql
+
+  SELECT * FROM citus_shards;
+
+::
+
+  .
+   shardid | table_name | citus_table_type | colocation_id | nodename  | nodeport | shard_size | shard_size_bytes
+  ---------+------------+------------------+---------------+-----------+----------+------------+------------------
+    102170 | dist       | distributed      |            34 | localhost |     9701 | 86 MB      |         90677248
```

Comment:
the output is a bit different now. see https://github.com/citusdata/citus/pull/4649.

Basically the shard_size is now in byte format that way it is easy to do operations on it (a bit hard when it is prettified).

Also the order is a bit different now